### DISCUSSION TITLE
Miami Track Layout

### DISCUSSION THREAD

**Discussion by pmkroeker** [theOehrly/Fast-F1#671] [Q&A] [answered]
Hi all,

I have been working through a series where I plot (to the best of the telemetry) overtaking spots on the tracks.

I have got a number of comments on the Miami track layout (a few other tracks as well, but most from Miami. I I have been using the methods on the documentation to draw and label the track, but the track looks a bit off compared to the online images. I have a picture of mine, and a link to the official track layout. Is there something odd about a few of the tracks that makes them look stretched? Or is there a better way to do it? Most tracks look pretty close.

My track layout:
![2024 Miami](https://github.com/user-attachments/assets/a185bf41-be7a-4740-876f-f5f8e2c030e4)

![Official](https://www.formula1.com/en/racing/2024/miami/circuit)

**Comment by Casper-Guo** [comment]
Hard to say without looking at your code. I suspect Matplotlib issue

**Reply by theOehrly** [reply, marked as answer]
Yes, most likely a Matplotlib issue related to axis scaling.
This might be helpful https://matplotlib.org/stable/gallery/subplots_axes_and_figures/axis_equal_demo.html

**Reply by pmkroeker** [reply]
yep, axis equal fixed it
